### PR TITLE
MF-219: Add conditions widget translations

### PR DESCRIPTION
**Pull request by denniskigen**
https://issues.openmrs.org/browse/MF-219

This PR:

- Adds translations strings for the components that make up the conditions widget.
- Improves the conditions widget test suite - statement coverage up to 90%.

### CODE REVIEW

File: src/widgets/conditions/conditions-detailed-summary.component.tsx [hunk 1]
```diff
@@ -31,28 +34,34 @@ export default function ConditionsDetailedSummary(
...
+          addComponent={ConditionsForm}
+          showComponent={() =>
+            openWorkspaceTab(ConditionsForm, `${t("Conditions Form")}`)
+          }
+        >
+          <table className={`omrs-type-body-regular ${styles.conditionTable}`}>
+            <thead>
+              <tr>
+                <td>
+                  <Trans i18nKey="condition_upper">CONDITION</Trans>
```
Comment (FlorianRappl) [inline src/widgets/conditions/conditions-detailed-summary.component.tsx]: I would not include *upper* strings. These transformations can be made via CSS (so I would rather specify a class here that uses `text-transform: uppercase`).

File: src/widgets/conditions/conditions-overview.component.tsx [hunk 2]
```diff
@@ -32,57 +33,71 @@ export default function ConditionsOverview(props: ConditionsOverviewProps) {
...
+          <SummaryCardRow>
+            <SummaryCardRowContent>
               <HorizontalLabelValue
-                label={condition.resource.code.text}
-                labelStyles={{ fontWeight: 500 }}
-                value={dayjs(condition.resource.onsetDateTime).format(
-                  "MMM-YYYY"
-                )}
-                valueStyles={{ fontFamily: "Work Sans" }}
+                label="Active Conditions"
```
Comment (FlorianRappl) [inline src/widgets/conditions/conditions-overview.component.tsx]: Should we translate the label?

File: src/widgets/shared-utils.tsx [hunk 3]
```diff
@@ -30,6 +30,11 @@ export function openWorkspaceTab<
   }
 }
 
+export const capitalize = s => {
```
Comment (FlorianRappl) [inline src/widgets/shared-utils.tsx]: Keep in mind that we have `lodash-es` as a dependency, which comes with this function.

**Review by FlorianRappl — COMMENTED** [review]
LGTM to so - great job!

File: src/widgets/conditions/conditions-overview.component.tsx (hunk 2 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-overview.component.tsx]: Yes!

File: src/widgets/conditions/conditions-detailed-summary.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-detailed-summary.component.tsx]: Good shout, amended.

File: src/widgets/shared-utils.tsx (hunk 3 above)
Comment (denniskigen) [reply src/widgets/shared-utils.tsx]: 🤙

**Review by FlorianRappl — APPROVED** [review]
LGTM! Great work @denniskigen :beers:

**Pull request merged**